Sources/RichTextKit/Attributes/RichTextLayoutManager.swift

```diff
@@ -0,0 +1,46 @@
+import UIKit
```

Comment:
Will this also work for macOS?

Reply:
Now it will. :) Just need to rebase and align the #ifdefs